networks/rococo/parachain/energywebx.json

```
@@ -0,0 +1,20 @@
+{
+  "data": {
+    "ParaID": 4237,
+    "Project Name": "ewx",
+    "SS58 Prefix": "",
+    "Logo": "../../assets/images/ewx.png",
+    "Cover": "",
+    "Description (en)": "Energy web X",
```

I think `Web` should be capitalized. This is how it is on our website for example
```suggestion
    "Description (en)": "Energy Web X",
```